aea/cli/add.py

```diff
@@ -48,16 +48,18 @@ def add(ctx: Context, registry):
 
 def _find_connection_locally(ctx, connection_public_id, click_context):
     # check that the provided path points to a proper connection directory -> look for connection.yaml file.
-    # first check in aea dir
+    # first check in registry
     connection_name = connection_public_id.name
     registry_path = ctx.agent_config.registry_path
-    connection_configuration_filepath = Path(os.path.join(registry_path, "connections", connection_name, DEFAULT_CONNECTION_CONFIG_FILE))
+    package_path = Path(registry_path, connection_public_id.author, "connections", connection_name)
+    connection_configuration_filepath = package_path / DEFAULT_CONNECTION_CONFIG_FILE
```

Comment:
Is this recommended style, never seen it before?

Reply:
yes good idea, will do

Reply:
I don't know if it is recommended, but I love that syntactic sugar :-) 

of course it requires that the left operand is an instance of `pathlib.Path`, and the right operand can be both `Path`, `str` or any other type accepted by the constructor of `Path`.

You can see it equivalent as the following expression:
`Path(package_path, DEFAULT_CONNECTION_CONFIG_FILE)`

Reply:
Yeah, it looks neat. But when I first saw it I thought it's some maths operation :D